.test-infra/jenkins/PythonTestProperties.groovy

```diff
@@ -23,8 +23,14 @@ class PythonTestProperties {
     LOWEST_SUPPORTED,
     HIGHEST_SUPPORTED
   ]
+  final static List<String> ALL_SUPPORTED_VERSIONS = [
+    LOWEST_SUPPORTED,
```

Comment:
Let's spell out the versions and if possible assign LOWEST_SUPPORTED=ALL_SUPPORTED_VERSIONS[0], 
HIGHEST_SUPPORTED=ALL_SUPPORTED_VERSIONS[-1] 
(I'm not sure of the exact syntax).